scripts/prepare-db-for-emoji.pl

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/perl
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+use strict;
+use warnings;
+use 5.10.1;
+
+use File::Basename;
+use File::Spec;
+BEGIN {
+    require lib;
+    my $dir = File::Spec->rel2abs(File::Spec->catdir(dirname(__FILE__), ".."));
+    lib->import($dir, File::Spec->catdir($dir, "lib"), File::Spec->catdir($dir, qw(local lib perl5)));
+    chdir $dir;
+}
+
+use Bugzilla;
+use Bugzilla::Constants;
+
+BEGIN { Bugzilla->extensions }
+
+# set Bugzilla usage mode to USAGE_MODE_CMDLINE
+Bugzilla->usage_mode(USAGE_MODE_CMDLINE);
+
+my $dbh = Bugzilla->dbh;
+
+my %mysql_var = map { @$_ } @{ $dbh->selectall_arrayref(q(SHOW GLOBAL VARIABLES LIKE 'innodb_%')) };
+my %wanted = (
+    innodb_file_format    => 'Barracuda',
```

Comment:
Why specifically Barracuda?

Reply:
Only Barracuda supports Dynamic row format.
Antelope is the default in 5.5/5.6 and only supports Compact.